src/index.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable */
```

Comment:
While you did an amazing job working on the project requirements I noticed you disabled eslint in this given file. It is against microverse policy to disable linters because linters help us write clean code it is also not good practice to disable linters so it would be nice if you enabled eslint in this file. You may want to consider implementing this in your Home.js, Calculator.js and button.js  files aswel